src/components/Card.js

```diff
@@ -38,9 +48,17 @@ export default class Card {
     this._likeButton.classList.toggle("card__like-button_active");
   }
 
-  _handleCardDelete() {
-    this._element.remove();
-    this._element = null;
+  _handleTrashClick() {
+    this._deleteCardModal.classList.add("modal_opened");
+    document.addEventListener("keydown", this._handleEscClose);
```

Comment:
Please, delete it from `Card.js` because any modal should be created in `index.js` using a class extended from `Popup`. There are `open` and `_handleEscClose` methods for that


`_handleTrashClick` is not needed here because you reassign it in the constructor in line 15

Reply:
* My remark hasn't been corrected https://github.com/ExpiredSunshine/se_project_aroundtheus/pull/1#discussion_r1946087044